src/hooks/useBookmarkAutoPronounce.js

```diff
@@ -33,7 +46,7 @@ export default function useBookmarkAutoPronounce() {
   }
 
   return [
-    currentPronouncingState,
+    autoPronounceBookmark,
     currentPronouncingString,
     cyclePronounciationState,
```

Comment:
The smallest ask ever: could we call this `togglePronounciationState`? It would be slightly clearer I think. Although I understand the current name too. And I see that theoretically it's more general than toggle. Although in practice, with this feature it will always be toggle... But you choose :)

Reply:
lets call it `toggleAutoPronounceState` so it's aligned with the place where we destructure it in NextNavigation!

Reply:
This was only because initially we did have it also had a Session-Only but yes, for now let's call it toggle - I will make a commit with the change